frontend/src/components/invoice/invoice-edit/EditInvoice.tsx

```diff
@@ -102,6 +103,7 @@ const EditInvoice = () => {
                   variant="link"
                 />
                 {initInvoice._id && <DownloadInvoiceButton invoice={initInvoice} />}
+                  {storeInvoice?._id && !storeInvoice?.isQuotation && <InvoiceDownloadIcon invoice={invoice} fileType='xml' styleOption={{color: '#0062cc', marginLeft: 20}} />}
```

Comment:
`styleOption`: Het is typisch conventie om voor dergelijke dingen om de React property naam te gebruiken. (max-width --> maxWidth), in dit geval dus gewoon `style`

Het voordeel is dat het direct duidelijk om een "echt" html attribute gaat en je kan dan ook makkelijk gaan spreaden:

```js
const {invoice, fileType, ...rest} = props
return <InvoiceDownloadIcon {...rest} />

// om dan uiteindelijk ergens te hebben
return <Icon {...props} />
```

ie minder mapping ala styleOption->style

Reply:
Aangepast